starter-core/src/main/java/io/micronaut/starter/feature/aws/template/cdkappstack.rocker.raw

```diff
@@ -151,10 +156,18 @@ public class AppStack extends Stack {
                 .handler(function)
                 .build();
 }
+@if (features.contains("amazon-api-gateway-http")) {
+        HttpLambdaIntegration integration = HttpLambdaIntegration.Builder.create("HttpLambdaIntegration", function)
+                .payloadFormatVersion(VERSION_1_0)
+                .build();
+        HttpApi api = HttpApi.Builder.create(this, "micronaut-function-api")
```

Comment:
can you use `functionId` argument instead of `"micronaut-function-api"`

Reply:
@Sergio I think this needs to be `apiId` function argument, not `functionId`; i.e. same as it is for the REST variant, right?

apiId = "micronaut-function-api"
functionId = "micronaut-function"

You example code in the Issue description has "micronaut-function-api"

```java
        HttpApi api = HttpApi.Builder.create(this, "micronaut-function-api")
                .defaultIntegration(integration)
                .build();
```

Reply:
yes, `apiId`